.github/workflows/ci.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
         ruby-version:
         - '3.0'
         - '3.1'
+        - '3.2'
+        - '3.3'
```

Comment:
Just offering my thoughts on the plugins and core... I think we should be supporting 3.0-3.3 in plugins until we drop 3.0 in the Gemfile.  I don't think we need to test all of those versions in plugins/core as I think it's fine to test only 3.0, 3.1, and 3.3 with the intention of dropping 3.0 as soon as we have smoke tested appliances and pods with rebuilt rpms against 3.3.

Repositories that don't load the core application (aka not core and not a plugin/engine) can test as many rubies and rails versions as make sense based on their usage.  Things tied somewhat to the core rails app without loading it should stray closer to the rails app's dependencies (pg ha admin, logical replication, etc.) while other gems that are completely independent should have their own ruby/rails dependencies (ancestry, default_value_for, optimist, etc.).